packages/slice-machine/pages/changes.tsx

```diff
@@ -83,8 +93,22 @@ const changes: React.FunctionComponent = () => {
           MainBreadcrumb={<Text ml={2}>Changes</Text>}
           breadrumbHref="/changes"
         />
-        <TextWithGreyBackground text={"Custom Types X"} />
-        <TextWithGreyBackground text={"Slices X"} />
+        <ChangesSectionHeader text={"Custom Types"} amount={0} />
```

Comment:
`amount = 0` ?

Reply:
that's just hard coded, we haven't taken the ticket to render the custom types in that section yet. When we do we'll have a list on unsynced CTs and we can use the length of it as the `amount`

Reply:
ok